crates/ra_ide/src/hover.rs

```diff
@@ -484,9 +772,14 @@ fn main() { let foo_test = fo<|>o(); }
         "#,
             expect![[r#"
                 *foo*
-                ```rust
+
+                ````rust
+                test
+                ````
+
+                ````rust
                 pub fn foo<'a, T: AsRef<str>>(b: &'a T) -> &'a str
-                ```
+                ````
```

Comment:
Why do we need four backtics now? This makes diff hard to read

Reply:
This occurs because it's how `pulldown_cmark_to_cmark` renders code blocks. It's probably worth a PR to that crate to make it configurable, I can try and do that before this is merged if you want.

Reply:
https://github.com/Byron/pulldown-cmark-to-cmark/issues/17